src/odb/src/db/dbModule.h

```diff
@@ -82,6 +84,11 @@ class _dbModule : public _dbObject
 
   // User Code Begin Fields
   // custom iterator for traversing ports
+  // fast access
+  std::unordered_map<std::string, uint> _dbinst_hash;
+  std::unordered_map<std::string, uint> _modinst_hash;
+  std::unordered_map<std::string, uint> _modbterm_hash;
+  std::unordered_map<std::string, uint> _modnet_hash;
```

Comment:
Instead of uint can you use the appropriately typed dbId ?

Reply:
done